Configure jumpable characters

oalejel opened the pull request:
Finally allows users to add jumpable characters in the settings view!

oalejel committed:
Configure jumpable characters

Finally allows users to add jumpable characters in the settings view!

oalejel commented:
IMPORTANT!!! Please make sure to update the version number!!!!!

telzul commented:
yeah i have seen that :) but as i understand the publishing process does this automatically:
http://flight-manual.atom.io/hacking-atom/sections/publishing/

oalejel commented:
Wow I had no idea! Thanks for the great package, it really makes things faster!

Omar

> On Aug 29, 2017, at 12:08 PM, Robert <notifications@github.com> wrote:
> 
> yeah i have seen that :) but as i understand the publishing process does this automatically:
> http://flight-manual.atom.io/hacking-atom/sections/publishing/ <http://flight-manual.atom.io/hacking-atom/sections/publishing/>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/telzul/atom-bracket-close-jump/pull/3#issuecomment-325713698>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGY1VlCq9PaL0MsqdfgdTP4P4GimLV0Oks5sdDeXgaJpZM4PGJ5p>.
>

telzul commented:
Cool that it helps you! As i understand you came from Xcode and this was some kind of default behaviour? I was missing it in atom coming from eclipse. 

Sorry merging takes so long, i just want to try it out, but i am not using atom anymore so first i am setting it up and try to figure out how to use your branch. :)

oalejel commented:
Precisely. Xcode’s intuitive bracket skipping with a tab was super useful, and it felt wrong not to have the feature in atom. 

Yes, please make sure that the branch works on your end. I haven’t done a test where I uninstall and reinstall the package, so your check should be better verification.

Thanks a ton! If something goes wrong for some reason, please let me know!

Omar

> On Aug 29, 2017, at 12:13 PM, Robert <notifications@github.com> wrote:
> 
> Cool that it helps you! As i understand you came from Xcode and this was some kind of default behaviour? I was missing it in atom coming from eclipse.
> 
> Sorry merging takes so long, i just want to try it out, but i am not using atom anymore so first i am setting it up and try to figure out how to use your branch. :)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/telzul/atom-bracket-close-jump/pull/3#issuecomment-325715103>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGY1VvBShPLDkWTrxj3rKUZdy5TTRpQaks5sdDi9gaJpZM4PGJ5p>.
>

oalejel commented:
Quick modification:
I realized that the config schema name for the jumpable characters array should probably be called “jumpChars” instead of “autocompleteCharacters.” If you make this name change, please also change the word “bracket-close-jump.autocompleteCharacters” on line 34 of bracket-close-jump.coffee to “bracket-close-jump.jumpChars”.

Sorry for the ambiguous variable name!


> On Aug 29, 2017, at 12:13 PM, Robert <notifications@github.com> wrote:
> 
> Cool that it helps you! As i understand you came from Xcode and this was some kind of default behaviour? I was missing it in atom coming from eclipse.
> 
> Sorry merging takes so long, i just want to try it out, but i am not using atom anymore so first i am setting it up and try to figure out how to use your branch. :)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/telzul/atom-bracket-close-jump/pull/3#issuecomment-325715103>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGY1VvBShPLDkWTrxj3rKUZdy5TTRpQaks5sdDi9gaJpZM4PGJ5p>.
>

telzul committed:
rename field for characters

telzul committed:
rename field for jump characters

telzul commented:
ok. it is published now. can you try it out?

oalejel commented:
Will do. Thanks 
> On Aug 29, 2017, at 1:06 PM, Robert <notifications@github.com> wrote:
> 
> ok. it is published now. can you try it out?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/telzul/atom-bracket-close-jump/pull/3#issuecomment-325730107>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGY1Vo7ucCYe-KEiuKt_VNuMDIZlNxbJks5sdEUZgaJpZM4PGJ5p>.
>

oalejel commented:
The editor isn’t showing the updated Readme file, and for some reason, Atom also shows 3 text fields to input jumpable characters. I’ll wait a few minutes to see if it just needs to update..

> On Aug 29, 2017, at 1:06 PM, Omar Al-Ejel <alejel.omar@gmail.com> wrote:
> 
> Will do. Thanks 
>> On Aug 29, 2017, at 1:06 PM, Robert <notifications@github.com <mailto:notifications@github.com>> wrote:
>> 
>> ok. it is published now. can you try it out?
>> 
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub <https://github.com/telzul/atom-bracket-close-jump/pull/3#issuecomment-325730107>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGY1Vo7ucCYe-KEiuKt_VNuMDIZlNxbJks5sdEUZgaJpZM4PGJ5p>.
>> 
>

telzul commented:
yeah i had that too. This is because of renaming the field. I think this is because they are in some storage in atom. Annoyingly the field did not get deleted when i removed the old version and updated to the new.
I don't know what to do about that :|
I installed the package on another computer, there everything looks fine. Only one settings field.
The updated readme also shows up on both computers.

oalejel commented:
Ok cool. I just hope that other people updating the package don’t have the conflict between the atom local storage and the updated fields. I don’t think they’ll have the problem, since their packages won’t have the old field in the first place. I think this should be fine!

Thanks a ton!


> On Aug 29, 2017, at 1:21 PM, Robert <notifications@github.com> wrote:
> 
> yeah i had that too. This is because of renaming the field. I think this is because they are in some storage in atom. Annoyingly the field did not get deleted when i removed the old version and updated to the new.
> I don't know what to do about that :|
> I installed the package on another computer, there everything looks fine. Only one settings field.
> The updated readme also shows up on both computers.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/telzul/atom-bracket-close-jump/pull/3#issuecomment-325734135>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGY1Vua2zhE-A52koBsFW0av6KUOsxcOks5sdEiJgaJpZM4PGJ5p>.
>

telzul commented:
Yeah that is what i hope too :)
Their versions should never have set the other two fields.

thank you for this awesome contribution!
